components/ProductCard.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div class="relative p-4 font-sans">
+    <NuxtLink :to="navigateUrl">
+      <img class="rounded-xl max-h-72 w-96"
+           :src="imageUrl"
+           :alt="title"
+           @error="handleImageError"
+      >
+      <div class="mt-4 max-w-72">
+        <span class="text-xl font-black">{{ title }}</span>
+      </div>
+    </NuxtLink>
+    <div class="mt-2">
+      <div class="flex items-center p-1">
+        <span v-for="rating in ratings" :key="rating">
+          <RatingSvg/>  
+        </span>
+        <span class="ml-2">{{ ratings }}.0</span>
+        <span class="px-1 font-thin text-gray-400"> ({{ soldCount }} Sell)</span>
+      </div>
+      <div class="p-1">
+        <span class="text-xl font-black">{{ price }} Price</span>
+        <span class="p-2 text-xl font-thin text-gray-400 line-through">{{ salePrice }}</span>
+        <span class="p-1 mx-1 text-sm font-thin text-white bg-red-600 rounded w-69 h-30">{{ priceOff }}</span>
```

Comment:
Can we wait for @GayaneKasparova's PR to merge, so that everyone will have correct tailwind classes? 
If possible, let's use correct colors defined in custom tailwind config file. Colors won't be changed so those can be modified already, font names will be changing with the PR.

Reply:
I am agree with You )

I just created pull request for simply view the changes
it's not final stage
When it will be final stage I will notify via teams channel